src/repositories/inMemory/user_repository.js

```diff
@@ -13,6 +13,12 @@ class inMemoryUserRepository
         return axios.get("http://localhost:3000/fixtures/userActivity.json")
         .catch(function (error) {console.log("Une erreur est survenue lors de la récupération des données concernant l'activité de l'utilisateur ayant l'id " + id + ". Message de l'erreur : " + error)});
     }
+
+    getUserAverageTimeSession(id)
+    {
+        return axios.get("http://localhost:3000/fixtures/averageSessionTime.json")
```

Comment:
Je ne veux pas voir http://localhost:3000/

Reply:
C'est noté, je change ça pour les prochaines PR